src/lambda/handler-runner/docker-runner/DockerContainer.js

```diff
@@ -174,11 +174,21 @@ export default class DockerContainer {
       })
     })
 
-    const { stdout: containerPortBinding } = await execa('docker', [
+    let containerPort
+    const { stdout: dockerPortOutput } = await execa('docker', [
       'port',
       containerId,
     ])
-    const containerPort = containerPortBinding.split(':')[1]
+    for (const line of dockerPortOutput.split('\n')) {
```

Comment:
I think it would be great to add a comment with explanation as to why it's parsed this way so when someone later looks at it will right away understand what's going on, otherwise it might be hard to comprehend

Reply:
Thank you for comment.
I added comments to help someone understand it.